### DISCUSSION TITLE
is Milkman supporting self-signed cert? I couldnot find it in settings.

### DISCUSSION THREAD

**Discussion by sherryli-1214** [warmuuh/milkman#183] [Q&A]
Milkman  is very strong tools. .However, i couldnot find how to set private key/cert by  self-signed? Thanks..

**Comment by warmuuh** [comment]
milkman doesnt provide any means to analyze/verify the certificates (yet), but you can switch off certificate validation to basically ignore any selfsigned certificates

![image](https://github.com/warmuuh/milkman/assets/1331628/c36ec65a-98fe-423b-96ae-8b25edf74466)
